packages/manager/apps/support/index.js

```diff
@@ -0,0 +1,20 @@
+import ovhUxManagerSupport from '@ovh-ux/manager-support';
+import ovhUxNgOvhOtrs from '@ovh-ux/ng-ovh-otrs';
```

Comment:
```suggestion
import ngOvhOtrs from '@ovh-ux/ng-ovh-otrs';
```
could do I think,

Reply:
It wouldn't match the pattern for the other packages :(